app/models/user_summary.rb

```diff
@@ -211,4 +195,13 @@ def user_counts(user_hash)
     end.compact.sort_by { |u| -u[:count] }
   end
 
+  def post_query
+    Post
+      .joins(:topic)
+      .includes(:topic)
+      .where('posts.post_type IN (?)', Topic.visible_post_types(@guardian&.user, false))
```

Comment:
```suggestion
      .where('posts.post_type IN (?)', Topic.visible_post_types(@guardian&.user, include_moderator_actions: false))
```

How about using a keyword argument here for the option instead? I find that it improves the readability of the code when we know what a boolean represents when passed to a method.